pages/content/amp-dev/documentation/guides-and-tutorials/start/create/prepare_for_discovery.md

```diff
@@ -1,38 +1,46 @@
 
-## Linking pages with `<link>`
+## Import the script
 
-To establish that a non-AMP page and an AMP page should be treated as being "paired" together, we add information about the AMP page to the non-AMP page and vice versa, in the form of `<link>` tags in the `<head>`.
+Unlike `amp-img`, the `amp-base-carousel` component is an extended component. Extended component logic is not included in the base AMP JS provided by the boilerplate code discussed earlier. This helps keep AMP pages light and lean by explicitly only loading JavaScript for components a page uses.
 
-Add the following to the non-AMP page:
+To use the `amp-base-carousel` component, we must import its script tag. Copy the tag below and place it into the head of the document.
 
-[sourcecode:html]
-<link rel="amphtml" href="https://www.example.com/url/to/amp/document.html">
-[/sourcecode]
+[tip type="note"]
+You can find the import script for each component at the top of its reference documentation.
+[/tip]
 
-And this to the AMP page:
+```html
+<script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel-0.1.js"></script>
+```
 
-[sourcecode:html]
-<link rel="canonical" href="https://www.example.com/url/to/full/document.html">
-[/sourcecode]
+Next, copy and paste another `<amp-img>` element under the first one, then wrap both inside `<amp-base-carousel>` tags, like below:
 
-## What if I only have one page?
+```html
+<amp-base-carousel width="680" height="410" layout="responsive">
+  <amp-img src="https://amp.dev/static/samples/img/amp.jpg" width="680" height="410" layout="responsive"></amp-img>
+  <amp-img src="https://amp.dev/static/samples/img/amp.jpg" width="680" height="410" layout="responsive"></amp-img>
+</amp-base-carousel>
+```
 
-If you only have one page, and that page is an AMP page, you must still add the canonical link to it, which will then simply point to itself:
+## Component attributes
 
-[sourcecode:html]
-<link rel="canonical" href="https://www.example.com/url/to/amp/document.html">
-[/sourcecode]
+You may notice we have familiar attributes, `width`, `height`, and `layout` defined. Like HTML, attributes are used everywhere in AMP. But, AMP uses additional attributes to customize components’ behavior. Some are [common element attributes](https://amp.dev/documentation/guides-and-tutorials/learn/common_attributes/?format=websites) and others are special to certain components. Let's add the `loop` attribute and set it to `true` and include more images:
 
-[tip type="read-on"]
-**READ ON –** Learn more about how Google finds AMP pages in [Google Search guidelines for AMP pages](https://support.google.com/webmasters/answer/6340290).
-[/tip]
+```html
+<amp-base-carousel loop="true"  width="450" height="300" layout="responsive">
+  <amp-img src="https://amp.dev/static/samples/img/amp.jpg" width="680" height="410" layout="responsive"></amp-img>
```

Comment:
Is it intentional that the first amp-img `src` is absolute and the rest are relative?